packages/playwright-test/src/reporters/html.ts

```diff
@@ -529,4 +537,16 @@ function isTextContentType(contentType: string) {
   return contentType.startsWith('text/') || contentType.startsWith('application/json');
 }
 
+function metadata(): Metadata {
+  const githubJobLink = process.env.GITHUB_SERVER_URL ? `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}` : undefined;
+  const gitlabJobLink = process.env.CI_JOB_URL;
+  const jenkinsJobLink = process.env.JOB_URL;
+
+  return {
+    generatedAt: Date.now(),
+    title: process.env.PLAYWRIGHT_HTML_REPORT_TITLE || 'Playwright Test Report',
```

Comment:
Environment variables are used (instead of JS reporter options) since (1) it's nice to be able to easily call git from within your shell:

```
PLAYWRIGHT_HTML_REPORT_TITLE="$(git log --pretty=format:'%h: %s' -n 1)" npm run ctest
```

(2) `playwright.config.ts` reporter options do not work nicely with command line `--reporter` flag.